lib/ansible/modules/cloud/azure/azure_rm_virtualnetworkpeering_facts.py

```diff
@@ -56,61 +56,74 @@
 
 RETURN = '''
 vnetpeerings:
-    description: A list of Virtual Network Peering facts.
+    description:
+        - A list of Virtual Network Peering facts.
     returned: always
     type: complex
     contains:
         id:
-            description: Id of current Virtual Network peering.
+            description: ID of current Virtual Network peering.
             returned: always
             type: str
             sample:
                 "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVnet/virtualNetworkPeerings/peer1"
         name:
-            description: Name of Virtual Network peering.
+            description:
+                - Name of Virtual Network peering.
             returned: always
             type: str
             sample: myPeering
         remote_virtual_network:
-            description: Id of remote Virtual Network to be peered to.
+            description:
+                - ID of remote Virtual Network to be peered to.
             returned: always
             type: str
             sample: /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVnet2
         remote_address_space:
-            description: The reference of the remote Virtual Network address space.
+            description:
+                - The reference of the remote Virtual Network address space.
             type: complex
+            returned: always
             contains:
                 address_prefixes:
-                    description: A list of address blocks reserved for this Virtual Network in CIDR notation.
+                    description:
+                        - A list of address blocks reserved for this Virtual Network in CIDR notation.
+                    returned: always
                     type: list
                     sample: 10.1.0.0/16
         peering_state:
-            description: The status of the virtual network peering.
+            description:
+                - The status of the virtual network peering.
```

Comment:
Since the name of the parameter uses 'state', I'd use it in the description as well, to make things consistent.
```suggestion
                - The state of the virtual network peering.
```